src/engine/Operation.cpp

```diff
@@ -213,17 +214,16 @@ void Operation::checkAbortion() const {
 
 // ______________________________________________________________________
 
-void Operation::checkAbortion(const auto& detailSupplier) const {
+void Operation::checkAbortion(const std::invocable auto& detailSupplier) const {
```

Comment:
Shouldn't it return something stringlike so `invocable_r_r<string_view>` (pseudoconcept, look up the exact syntax yourself). Also add the same concept to the throwIfAborted function.